consensus/dummy/consensus.go

```diff
@@ -139,29 +139,22 @@ func (eng *DummyEngine) verifyHeaderGasFields(config *params.ChainConfig, header
 		}
 	}
 
-	if !config.IsApricotPhase3(header.Time) {
-		// Verify BaseFee is not present before AP3
-		if header.BaseFee != nil {
-			return fmt.Errorf("invalid baseFee before fork: have %d, want <nil>", header.BaseFee)
-		}
-	} else {
-		// Verify header.Extra matches the expected value.
-		expectedExtraPrefix, err := CalcExtraPrefix(config, parent, header.Time)
-		if err != nil {
-			return fmt.Errorf("failed to calculate extra prefix: %w", err)
-		}
-		if !bytes.HasPrefix(header.Extra, expectedExtraPrefix) {
-			return fmt.Errorf("expected header.Extra to have prefix: %x, found %x", expectedExtraPrefix, header.Extra)
-		}
+	// Verify header.Extra matches the expected value.
+	expectedExtraPrefix, err := CalcExtraPrefix(config, parent, header.Time)
+	if err != nil {
+		return fmt.Errorf("failed to calculate extra prefix: %w", err)
+	}
+	if !bytes.HasPrefix(header.Extra, expectedExtraPrefix) {
+		return fmt.Errorf("expected header.Extra to have prefix: %x, found %x", expectedExtraPrefix, header.Extra)
+	}
 
-		// Verify header.BaseFee matches the expected value.
-		expectedBaseFee, err := CalcBaseFee(config, parent, header.Time)
-		if err != nil {
-			return fmt.Errorf("failed to calculate base fee: %w", err)
-		}
-		if !utils.BigEqual(header.BaseFee, expectedBaseFee) {
-			return fmt.Errorf("expected base fee (%d), found (%d)", expectedBaseFee, header.BaseFee)
-		}
+	// Verify header.BaseFee matches the expected value.
+	expectedBaseFee, err := CalcBaseFee(config, parent, header.Time)
+	if err != nil {
+		return fmt.Errorf("failed to calculate base fee: %w", err)
+	}
+	if !utils.BigEqual(header.BaseFee, expectedBaseFee) {
+		return fmt.Errorf("expected base fee (%d), found (%d)", expectedBaseFee, header.BaseFee)
```

Comment:
nit can we stop using parentheses where it doesn't make much sense to use them 🙏  Unless there is a reason but it itches my eyes 🤩 

```suggestion
		return fmt.Errorf("expected base fee %d, found %d", expectedBaseFee, header.BaseFee)
```